includes/amp-enhancements/class-wp-gdpr-cookie-notice-performance.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * AMP improvements for WP GDPR Cookie Notice.
+ *
+ * @package Newspack
+ */
+
+use Felix_Arntz\WP_GDPR_Cookie_Notice\Shortcodes\WordPress_Shortcode_Parser;
+use Felix_Arntz\WP_GDPR_Cookie_Notice\Settings\Plugin_Option_Reader;
+
+namespace Newspack;
```

Comment:
This one isn't triggering a fatal for me, for whatever reason, but my understanding is that namespace declarations should always come first, right?

Reply:
Yep, you're correct. This bit wasn't triggering an error because those namespaces aren't actually used any more (they were from an earlier iteration). :) I've removed them in https://github.com/Automattic/newspack-plugin/pull/622/commits/6db2a0c38e5543f3f972a9bb0acb441a6940a53e

Reply:
Makes sense, thanks!